tests/foreman/api/test_multiple_paths.py

```diff
@@ -474,6 +475,10 @@ def test_delete_and_get(self, entity):
             self.fail(err)
         logger.info('test_delete_and_get path: {0}'.format(entity_n.path()))
         entity_n.delete()
+        if entity == entities.Repository:
```

Comment:
I think would be better to do this workaround if the entity is Repository and BZ #1166365 is open.

```python
if entity == entities.Repository and bz_bug_is_open(1166365):
```